packages/contract-wrappers/test/revert_validation_test.ts

```diff
@@ -112,7 +107,7 @@ describe('Revert Validation ExchangeWrapper', () => {
                 makerTokenBalance,
             );
             await web3Wrapper.awaitTransactionSuccessAsync(txHash, constants.AWAIT_TRANSACTION_MINED_MS);
-            expect(
+            return expect(
```

Comment:
This is subtle, but there was a race condition here before adding the `return`.

Reply:
Can we add a TSLint rule for this? If not now, let's add a task to the backlog.

Reply:
➕ Good idea. I believe we already have a check for unhandled promise rejections. The problem is that `expect().to.be.rejectedWith` doesn't return a real `Promise`. Instead it returns a `Chai.PromisedAssertion`. There must be some way to check for it.